_pages/policies/tech-policies/security-incidents.md

```diff
@@ -13,20 +13,18 @@ See GSA’s Insite: [Report IT Security Incidents or Suspicious Activity](https:
 If you detect any unusual or suspicious activity on your computer, DO NOT turn off your computer. By turning off the computer, valuable evidence may be lost. For questions about GSA’s Incident Response Program, contact the GSA Incident Response (IR) Team at [gsa-ir@gsa.gov](mailto:gsa-ir@gsa.gov).
 
 ## Reporting phishing emails
-If you receive a phishing email, follow these steps to report to GSA IT:
+If you receive a phishing email, follow these steps:
```

Comment:
What if we change this a bit to say "If you receive a suspicious email" It feels like a lower bar and people may be more likely to report.

Reply:
Sounds great to me!